### DISCUSSION TITLE
Does anyone know what styles and code is required from Boostrap

### DISCUSSION THREAD

**Discussion by GregObleshchuk** [snapappointments/bootstrap-select#2875] [General]
Hi All,
It doesn't look like Bootstrap 5 will be supported any time soon.  I know I can load bootstrap 3 in my project as well as Bootstrap 5 but I was wondering if anyone knows what out of bootstrap 3 is required to format and make the select work?
My idea is that I could just pull that our of boostrap and use that.

cheers

**Comment by mattymatty76** [comment]
try this, I'm using it in a my project and the select works very well

https://gist.github.com/mattymatty76/c996d3b77f298b2ec133be59992df9d4

**Reply by GregObleshchuk** [reply]
Thanks , that does work well

**Reply by mattymatty76** [reply]
Happy to be helpful,
remember that if you have more than 600 items in a select, you need to extend virtualScroll from 600 (default value) to 100000 or alternatively disable it

https://github.com/snapappointments/bootstrap-select/issues/2524

**Reply by zhihui-l** [reply]
Hi, what css do you use to be compatible with gamma version bootstrap-select
